tests/sentry/profiles/test_device.py

```diff
@@ -0,0 +1,31 @@
+from unittest import TestCase
+
+from sentry.profiles.device import GIB, DeviceClass, classify_device
+
+
+class ProfilesDeviceTest(TestCase):
```

Comment:
These kinds of tests might be cleaner to write using `@pytest.mark.parametrize` but not a blocker on the PR